Lib/test/test_builtin.py

```diff
@@ -2403,6 +2423,7 @@ def child(wpipe):
     def test_input_tty(self):
         # Test input() functionality when wired to a tty (the code path
         # is different and invokes GNU readline if available).
+        return
```

Comment:
Oops, thanks. (I put that in while testing locally because otherwise the test hangs for me; should have removed it before committing.)
```suggestion
```